CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [v0.16.1](https://github.com/feast-dev/feast/tree/v0.16.1) (2021-12-10)
+
+[Full Changelog](https://github.com/feast-dev/feast/compare/v0.16.0...v0.16.1)
+
+**Fixed bugs:**
+
+- Bump log4j-api from 2.13.2 to 2.15.0 in /java [\#2133](https://github.com/feast-dev/feast/pull/2133) ([dependabot[bot]](https://github.com/apps/dependabot))
```

Comment:
Do you wanna include #2134 as well?

Reply:
im confused because that PR... doesn't introduce any change over #2133. It's the same change in the same line. Going to ignore for now, but it's an empty commit

Reply:
Oh lol. yeah you can ignore it